token-holders/how-to-stake.md

```diff
@@ -0,0 +1,40 @@
+# The staking UI
+
+{% hint style="info" %}
+Staked tokens can't be withdrawn immediately, make sure you understand the [rules](staking-rules.md) before staking your tokens.
+{% endhint %}
+
+First, go to the [staking frontend](https://staking.pyth.network/) and connect your wallet
+
+Once you connect your wallet, you'll be presented with the state of your current stake account.
+
+The tokens in the current connected wallet can be seen next to `Balance`.
+
+A user can have 3 types of tokens in the staking program:
+- Locked : these are tokens that are staked, they can be unlocked by going to the `Unlock` tab
+- Unlocked : these tokens are free to be withdrawn in the `Withdraw` tab
+- Unvested : these tokens are subject to a vesting schedule, they can only be withdrawn once they vest. A user can see when is the next vesting event by clicking on `Unvested`. Additionally any owner of unvested tokens can choose to stake them, making them eligible to vote.
```

Comment:
looking at the other pages, I think we want to standardize how we use colons -- so far there doesn't seem to be a space before them so maybe we can follow that standard?
e.g. Locked: instead of Locked :

Reply:
same for other pages in this PR that uses colons

Reply:
I suggest using bolding here and making the first word part of the sentence like this:

**Locked** tokens are staked ...